crates/project/src/environment.rs

```diff
@@ -71,8 +70,10 @@ impl ProjectEnvironment {
     }
 
     /// Returns all errors with associated worktree ids
```

Comment:
Comment is wrong, no?

Reply:
Yeah, I probably should add that we are returning the pairs of worktree_id and associated environment error messages